### PR TITLE
testing/nginx-naxsi: modernize and upgrade to 0.56

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by andypost** [comment]
Let's get @itoffshore approval

**Comment by itoffshore** [comment]
yes please go ahead

File: testing/nginx-naxsi/APKBUILD [hunk 1]
```diff
@@ -42,7 +42,7 @@ subpackages="$pkgname-doc"
 source="https://nginx.org/download/$_pkgname-$pkgver.tar.gz
 	naxsi-$_ngx_naxsi_ver.tar.gz::https://github.com/nbs-system/$_ngx_naxsi_name/archive/$_ngx_naxsi_ver.tar.gz
 	ngx_cache_purge-$_ngx_cache_purge_ver.tar.gz::https://github.com/itoffshore/$_ngx_cache_purge_name/archive/v$_ngx_cache_purge_ver.tar.gz
-	upstream-fair-$_ngx_upstream_fair_ver.tar.gz::https://github.com/itoffshore/$_ngx_upstream_fair_name/archive/v$_ngx_upstream_fair_ver.tar.gz
+	upstream-fair-$_ngx_upstream_fair_ver.tar.gz::https://github.com/itoffshore/$_ngx_upstream_fair_name/archive/$_ngx_upstream_fair_ver.tar.gz
```
Comment (andypost) [inline testing/nginx-naxsi/APKBUILD]: I'd suggest to get rid of `$_ngx_upstream_fair_name` variables in URLs to make easy navigate to source - could be follow-up material

**Comment by andypost** [comment]
Commited 8bc0953c

**Pull request closed without merging**